### ISSUE TITLE
The LogDepthL1loss code

### ISSUE THREAD

**Issue by eecoder-dyf** [saic-vul/saic_depth_completion#9]
In `saic_depth_completion\metrics\absolute.py`, class `LogDepthL1loss`, Line 35, 
```python
diff = torch.abs(torch.log(gt[mask]) - pred[mask])
```
Is there something mistake? Why use |log(gt)-pred|?
I think the code might be
```
diff = torch.abs(torch.log(gt[mask])-torch.log(pred[mask]))
```
Is this correct?

**Comment by DiTo97** [comment]
Hi @eecoder-dyf,

The code is correct as-is, since the model is already tasked with predicting the logarithmic depth (see [dm_lrn.py#L28](https://github.com/SamsungLabs/saic_depth_completion/blob/master/saic_depth_completion/modeling/dm_lrn.py#L28). ).

Therefore, we only have to apply the logarithm to the ground truth depth to directly compute the L1 loss.
